pkg/cmd/pr/merge/merge.go

```diff
@@ -216,6 +216,7 @@ func mergeRun(opts *MergeOptions) error {
 	deleteBranch := opts.DeleteBranch
 	crossRepoPR := pr.HeadRepositoryOwner.Login != baseRepo.RepoOwner()
 	autoMerge := opts.AutoMergeEnable && !isImmediatelyMergeable(pr.MergeStateStatus)
+	localBranchExists := git.HasLocalBranch(pr.HeadRefName)
```

Comment:
Unsure if `pr merge` did this consistently, but note that we should never access the local git directory if the `-R/--repo` flag was passed. Please keep that in mind when adding new git calls.